src/kedro_sphinx_theme/theme/kedro-sphinx-theme/layout.html

```diff
@@ -72,13 +72,14 @@
 
     // Iterate over each link to check if the current path includes the link's unique ID
     navLinks.forEach(function(link) {
```

Comment:
@rashidakanchwala I noticed there are 6 links in `navLinks` instead of 3, as 3 links is hidden.